crates/bevy_ecs/src/system/system_registry.rs

```diff
@@ -43,6 +43,22 @@ pub struct SystemId<I = (), O = ()> {
     pub(crate) marker: std::marker::PhantomData<fn(I) -> O>,
 }
 
+impl<I, O> SystemId<I, O> {
+    /// Get the underlying [`Entity`] value of a [`SystemId`].
+    pub fn entity(self) -> Entity {
+        self.entity
+    }
+
+    /// Create [`SystemId`] from an [`Entity`].
+    /// The entity must be a system & the `I` + `O` types must be correct to use this for running.
```

Comment:
This could use a bit more clarification. Maybe a doc example about how this could  be useful?

Reply:
This is a quite abstract & can't come up with an example that'd fit in a doc string.  The usecase I have in mind is with using components & entities to do key bindings. The simplest I could think of is propagation via hierarchy but you can't spawn systems manually right now so that wouldn't make sense.

Reply:
Yep, that's great.